projects/vdk-plugins/vdk-server/src/vdk/plugin/server/server_plugin.py

```diff
@@ -37,7 +38,13 @@
     is_flag=True,
     help="Returns whether a local Control Service is currently installed.",
 )
-def server(install, uninstall, status):
+@click.option(
+    "-si",
+    "--server-id",
```

Comment:
Can you please update the description of the PR to mention what exactly the change is? I think it adds a bunch of config options and adds this flag, but it's not super obvious.

Reply:
Removed that. Thanks for pointing it out. It was not supposed to be in this change. It's part of my experiments for getting tests to work